### ISSUE TITLE
Data: able to extract tally of nomad data

### ISSUE THREAD

**Issue by engerm** [RagtagOpen/nomad#91]
Tally of:
- Unique rides
- \# of canvas days a volunteer was there
- \# of unique trips
- \# of locations to/from where
- \# where searching from a particular location

We need to be able to know how much volunteer power was sent

**Comment by flynnwastaken** [comment]
Important for us, campaigns will be keeping their own tallies which they will likely be willing to share and help us develop the fuller picture of our impact.

**Comment by iandees** [comment]
Admins can download CSV now.

![image](https://user-images.githubusercontent.com/261584/31578868-8a84f99a-b0ef-11e7-87b7-661a2119d559.png)

**Comment by kielni** [comment]
I don't think this is quite the same as the user contact info CSV, but maybe it's lower priority now that we have that.

**Comment by phildini** [comment]
So we now have a stats page, and I'm open for ideas of where we should take it.
